cli/cmd/accounts.go

```diff
@@ -57,33 +59,126 @@ var (
 			// TODO: Nicer overall formatting.
+
+			consensusAccount, err := c.Consensus().Staking().Account(ctx, ownerQuery)
+			cobra.CheckErr(err)
+
 			fmt.Printf("Address: %s\n", addr)
+			fmt.Printf("Nonce: %d\n", consensusAccount.General.Nonce)
 			fmt.Println()
 			fmt.Printf("=== CONSENSUS LAYER (%s) ===\n", npw.NetworkName)
 
-			consensusAccount, err := c.Consensus().Staking().Account(ctx, &staking.OwnerQuery{
-				Height: consensus.HeightLatest,
-				Owner:  addr.ConsensusAddress(),
-			})
+			outgoingDelegations, err := c.Consensus().Staking().DelegationInfosFor(ctx, ownerQuery)
+			cobra.CheckErr(err)
+			outgoingDebondingDelegations, err := c.Consensus().Staking().DebondingDelegationInfosFor(ctx, ownerQuery)
 			cobra.CheckErr(err)
 
-			fmt.Printf("Balance: %s\n", helpers.FormatConsensusDenomination(npw.Network, consensusAccount.General.Balance))
-			// TODO: Delegations.
-			// TODO: Allowances.
+			helpers.PrettyPrintAccountBalanceAndDelegationsFrom(
+				npw.Network,
+				addr,
+				consensusAccount.General,
+				outgoingDelegations,
+				outgoingDebondingDelegations,
+				"  ",
+				os.Stdout,
+			)
+			fmt.Println()
 
-			if npw.ParaTime != nil {
-				// Query runtime account when a paratime has been configured.
+			if len(consensusAccount.General.Allowances) > 0 {
+				fmt.Println("  Allowances for this Account:")
+				helpers.PrettyPrintAllowances(
+					npw.Network,
+					addr,
+					consensusAccount.General.Allowances,
+					"    ",
+					os.Stdout,
+				)
 				fmt.Println()
-				fmt.Printf("=== %s PARATIME ===\n", npw.ParaTimeName)
+			}
 
+			incomingDelegations, err := c.Consensus().Staking().DelegationsTo(ctx, ownerQuery)
+			cobra.CheckErr(err)
+			incomingDebondingDelegations, err := c.Consensus().Staking().DebondingDelegationsTo(ctx, ownerQuery)
+			cobra.CheckErr(err)
+
+			if len(incomingDelegations) > 0 {
+				fmt.Println("  Active Delegations to this Account:")
+				helpers.PrettyPrintDelegationsTo(
+					npw.Network,
+					addr,
+					consensusAccount.Escrow.Active,
+					incomingDelegations,
+					"    ",
+					os.Stdout,
+				)
+				fmt.Println()
+			}
+			if len(incomingDebondingDelegations) > 0 {
+				fmt.Println("  Debonding Delegations to this Account:")
+				helpers.PrettyPrintDelegationsTo(
+					npw.Network,
+					addr,
+					consensusAccount.Escrow.Debonding,
+					incomingDebondingDelegations,
+					"    ",
+					os.Stdout,
+				)
+				fmt.Println()
+			}
+
+			cs := consensusAccount.Escrow.CommissionSchedule
+			if len(cs.Rates) > 0 || len(cs.Bounds) > 0 {
+				fmt.Println("  Commission Schedule:")
+				cs.PrettyPrint(ctx, "    ", os.Stdout)
+				fmt.Println()
+			}
+
+			sa := consensusAccount.Escrow.StakeAccumulator
+			if len(sa.Claims) > 0 {
+				fmt.Println("  Stake Accumulator:")
+				sa.PrettyPrint(ctx, "    ", os.Stdout)
+				fmt.Println()
+			}
+
+			if npw.ParaTime != nil {
+				// I'm too smoothbrain to figure out how to convert from
+				// height to round, instead of complaining about this in
```

Comment:
Something like:
```golang
blk, err := c.Consensus().RootHash().GetLatestBlock(ctx, &RuntimeRequest{
    RuntimeID: npw.ParaTime.Namespace(),
    Height:    common.GetHeight(),
})
cobra.CheckErr(err)
round := blk.Header.Round

Reply:
`Error: rpc error: code = Unimplemented desc = unknown service oasis-core.RuntimeClient`

or

`Error: rpc error: code = Unimplemented desc = Only some methods are allowed on this proxy.`

Reply:
Oh well.